mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule RailwayUi.MixProject do
   def project do
     [
       app: :railway_ui,
-      version: "0.0.7",
+      version: "0.0.8",
```

Comment:
One question re versioning (https://semver.org/). I know we're not totally following this (we make all sort of feature upgrades in minor bumps), but is this backwards compatible? Or does railway needing 1.5 require the app it's running on to have 1.5 also? Maybe this is a 0.1.0 bump?

Reply:
That's a good point. I'll bump it to 0.1.0.